index.bs

```diff
@@ -1162,10 +1160,10 @@ requirements for a remote-playback-id.
 : sources
 :: The [=media resources=] that the controller has selected for playback
     on the receiver.  Each source must include a <{source/src|source URL}>
-    and should include an <{source/type|extended MIME type}>.
+    and should include an <{source/type|extended MIME type}>.  If empty,
```

Comment:
If it's allowed to be empty, isn't this a "may include"?

Reply:
Yes, it can be empty, but if the media element has one it should be included in the `remote-playback-start-request`.  Added some text to clarify.

Reply:
Sorry, thought that comment was about the mime type :)